inlang/source-code/sdk/src/v2/createMessageBundle.ts

```diff
@@ -0,0 +1,121 @@
+import {
+	LanguageTag,
+	Declaration,
+	MessageBundle,
+	Message,
+	Pattern,
+	Text,
+	Expression,
+} from "./types.js"
+
+export function createMessageBundle(
+	id: string,
+	localeText: Record<LanguageTag, string>,
+	options?: {
+		alias?: MessageBundle["alias"]
+		variableReferencePattern?: string[]
+	}
```

Comment:
Q: should we switch to passing an args object instead of regular function params?

Reply:
👍 changed.